lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -888,17 +885,17 @@ sv_vendor_axis_communications_set_attestation_report(signed_video_t *sv,
       sv->onvif = onvif_media_signing_create(sv_codec_to_ms_codec(sv->codec));
     }
   }
+  // Set the signing key pair if sign_data is available
```

Comment:
I still think this comment is unclear. If you ask any other person in our team. Do you think it can be easily understood? Can you describe it in common words rather in terms of variables, that is, without `sign_data`?